spec/features/visitor_sees_show_spec.rb

```diff
@@ -1,13 +1,14 @@
-require 'rails_helper'
```

Comment:
If this test is no longer used, please delete the file

Reply:
deleted file